hack/verify-generated-crd-code.sh

```diff
@@ -17,3 +17,13 @@
 HACK_DIR=$(dirname "${BASH_SOURCE}")
 
 ${HACK_DIR}/update-generated-crd-code.sh --verify-only
+
+# ensure no changes to generated CRDs
+if ! git diff --exit-code pkg/generated/crds/crds.go >/dev/null; then
+  # revert changes to state before running CRD generation to stay consistent
+  # with code-generator `--verify-only` option which discards generated changes
+  git checkout pkg/generated/crds
+
+  echo "run 'make update' to update generated CRDs"
```

Comment:
Let's make it a little clearer in this message that there's an error due to the CRDs being out of date, and the solution is 'make update'

Reply:
updated, basing it a bit off of the gofmt error message https://github.com/heptio/velero/blob/166565743f46e0bd254f5a41d80720a5198ce7ea/hack/update-fmt.sh#L46